pepper-apis/src/main/resources/db-changes/schema/DDP-4458-truncate-varchar-for-utf8mb4.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20200330-truncate-varchar-for-utf8mb4">
+        <modifyDataType tableName="umbrella_study" columnName="study_name" newDataType="varchar(200)"/>
+
+        <modifyDataType tableName="activity_validation" columnName="precondition_text" newDataType="varchar(5000)"/>
+        <modifyDataType tableName="activity_validation" columnName="expression_text" newDataType="varchar(5000)"/>
```

Comment:
Both of these column are in the same table, and along with other columns in here, will extend mysql's max row size. Originally at `varchar(10000)` each, these add up to 20,000 * 4 bytes is 80,000 bytes which is way over their limit of ~65KB.